ReactWindows/ReactNative/Views/Scroll/ReactScrollViewManager.cs

```diff
@@ -22,8 +23,25 @@ public class ReactScrollViewManager : ViewParentManager<ScrollViewer>
     {
         private const int CommandScrollTo = 1;
 
-        private readonly IDictionary<ScrollViewer, ScrollViewerData> _scrollViewerData =
-            new Dictionary<ScrollViewer, ScrollViewerData>();
+        /// <summary>
+        /// Attached property for ScrollViewerData
+        /// </summary>
+        public static readonly DependencyProperty ScrollViewerDataProperty =
+            DependencyProperty.RegisterAttached(
+                "ScrollViewerData",
+                typeof(ScrollViewerData),
```

Comment:
Besides minimizing the diff, any benefit to keeping the ScrollViewerData class intact rather than splitting it up into a bunch of attached properties on the ScrollViewer?